internal/service/adapters.go

```diff
@@ -115,14 +118,23 @@ func NewEventsForwarder(c config.Config) (ports.EventsForwarder, error) {
 func NewRuntimeKubernetes(c config.Config) (ports.Runtime, error) {
 	var masterURL string
 	var kubeConfigPath string
+	var qps int
+	var burst int
 
 	inCluster := c.GetBool(runtimeKubernetesInClusterPath)
-	if !inCluster {
+	if inCluster {
+		qps = c.GetInt(runtimeKubernetesQPS)
```

Comment:
If the configs are not there, it will default to GoLang's zero values (for ints is 0). If 0 is provided as a config value for the kubernetes client, it will default to `qps: 5` and `burst: 10` ([ref](https://github.com/kubernetes/client-go/blob/2371cfb50000e60d62c597f5c79eead80420203b/rest/config.go#L354)). That remains true for the latest version of the lib ([ref](https://github.com/kubernetes/client-go/blob/d99a76c51e14d805660d3d3e54a1d9f1920dc4a1/rest/config.go#L354)).

That was already the case for the older code, where we didn't provide values for QPS and Burst if running on InCluster mode. Do you think this change is still necessary?

Reply:
No. Thanks for the explanation.